crml/cennzx-spot/src/types.rs

```diff
@@ -13,91 +13,144 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
-//!
-//! CENNZX-SPOT Types
-//!
 use cennznet_primitives::types::AssetId;
-use core::convert::TryInto;
-use uint::construct_uint;
-construct_uint! {
-	/// 128-bit unsigned integer.
-	pub struct U128(2);
+use codec::{Decode, Encode, HasCompact};
+use core::{
+	convert::{From, Into, TryFrom},
+	marker::PhantomData,
+};
+use support::decl_error;
+
+pub use primitive_types::U256 as HighPrecisionUnsigned;
+pub use u128 as LowPrecisionUnsigned;
```

Comment:
These aliases are confusing to me because there's no reference to the actual types u128 and U256. Is there a reason why we can't use the original names?

Reply:
If we decide in the future that our low precision threshold would be 256 which is the case for Etherium and a likely change for us, we shouldn't need to go and change many places of the code.